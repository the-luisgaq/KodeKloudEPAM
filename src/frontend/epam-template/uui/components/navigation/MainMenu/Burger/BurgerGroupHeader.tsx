import * as React from 'react';
import cx from 'classnames';
import css from './BurgerGroupHeader.module.scss';

interface BurgerGroupHeaderProps {
    /*
    * Defines component caption.
    */
    caption: string;
}

export function BurgerGroupHeader(props: BurgerGroupHeaderProps) {
    return (
        <div className={ cx(css.root, css.groupHeader, 'uui-burger-group-header') }>
            <hr className={ css.line } />
            <span className={ css.groupName }>{props.caption}</span>
        </div>
    );
}
