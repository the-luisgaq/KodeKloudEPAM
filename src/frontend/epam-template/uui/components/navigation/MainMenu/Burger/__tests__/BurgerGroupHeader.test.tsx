import React from 'react';
import { BurgerGroupHeader } from '../BurgerGroupHeader';
import { render } from '@epam/uui-test-utils';

describe('BurgerSearch', () => {
    it('should be rendered correctly', () => {
        const { asFragment } = render(<BurgerGroupHeader caption="Test" />);
        expect(asFragment()).toMatchSnapshot();
    });
});
