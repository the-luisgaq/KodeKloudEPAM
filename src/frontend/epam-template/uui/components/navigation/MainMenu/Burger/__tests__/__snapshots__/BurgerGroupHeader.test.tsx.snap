// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BurgerSearch should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="root groupHeader uui-burger-group-header"
  >
    <hr
      class="line"
    />
    <span
      class="groupName"
    >
      Test
    </span>
  </div>
</DocumentFragment>
`;
